partcad/tests/unit/test_git_clone_retries.py

```
@@ -90,8 +91,15 @@
         raise git_error
     side_effect.counter = 0
 
+    def git_config(*args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for private function `git_config`

(ANN202)